### DISCUSSION TITLE
Darknet to ONNX converter

### DISCUSSION THREAD

**Discussion by ogoun** [AlexeyAB/darknet#8060] [Q&A]
Please, advice best convertor to onnx for darknet. 
I use next:
https://github.com/phanxuanduc1996/convert_yolo_weights
https://github.com/Tianxiaomo/pytorch-YOLOv4
https://github.com/WongKinYiu/PyTorch_YOLOv4

But with them accuracy is lost. And conversion of Yolov4-Scaled does not work.

**Comment by gitE0Z9** [comment]
This might sound like a long shot, but my workaround is using https://github.com/david8862/keras-YOLOv3-model-set
First convert from darknet to keras and then onnx.
Then I learned how to write my own conversion script to pytorch too.

You can give it a try!
